examples/dreamcast/raylib/tetris/src/game/game.cpp

```diff
@@ -0,0 +1,333 @@
+#include "game.h"
+#include "../constants/constants.h"
+#include <random>
+#include <kos.h>
+#include <dc/maple.h>
+#include <dc/maple/controller.h>
+
+// The below moves are in numpad notation because I can't understand them otherwise.
+const int Game::moves[15][2] = {
+    {1, 0},   // Move 2
+    {1, 1},   // Move 1
+    {0, 1},   // Move 4
+    {1, -1},  // Move 3
+    {0, -1},  // Move 6
+    {-1, 0},  // Move 8
+    {-1, 1},  // Move 7
+    {-1, -1}, // Move 9
+    {0, -2},  // Move 66 (stuck on a wall fallback)
+    {0, 2},   // Move 44 (stuck on a wall fallback)
+    {-2, 0},  // Move 88
+    {-2, 1},  // Move 87
+    {-2, -1}, // Move 89
+    {-2, 2},  // Move 77
+    {-2, -2}, // Move 99
+};
+
+Game::Game(){
+    grid = Grid();
+    blocks = GetAllBlocks();
+    currentBlock = GetRandomBlock();
+    nextBlock = GetRandomBlock();
+    prev_buttons = 0;
+    gameOver = false;
+    score = 0;
+    lastHeldMoveTime = 0.0;
+    vmuManager.resetImage();
+}
+
+// Game::~Game(){
```

Comment:
```suggestion
// Game::~Game(){
```
Logic was moved to `SfxManager`, no need to comment out. Just delete.